### PR TITLE
Stack of updates to docs in preparation for 1.0

### PR DESCRIPTION
**Pull request by yadudoc**

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -71,11 +71,14 @@ can be installed easily via `pip` using a pip extras option.
...
 
 * Amazon Web Services (Cloud) : `aws`
+* OAuth based SSH : `oauth_ssh`
+* Extreme Scale Executor (Supercomputing) : `extreme_scale`
 * Google Cloud : `google_cloud`
+* Python GSSAPI for SSH : `gssapi`
+* Azure (Cloud) : `azure`
 * Kubernetes : `kubernetes`
-* Extreme Scale Executor (Supercomputing) : `extreme_scale`
-* Logging monitoring data to a database: `monitoring`
```
Comment (benclifford) [inline docs/quickstart.rst:77]: `monitoring` is still an extra dependency target

File: docs/quickstart.rst [hunk 2]
```diff
@@ -71,11 +71,14 @@ can be installed easily via `pip` using a pip extras option.
...
-* Jetstream (NSF Cloud) : `jetstream`
+* Work Queue execution framework : `workqueue`
+
```
Comment (benclifford) [inline docs/quickstart.rst:81]: There is also a target for `gssapi`, I guess to do with the ssh channel?

Comment (yadudoc) [reply docs/quickstart.rst:81]: Yes, that's right.

File: docs/quickstart.rst (hunk 1 above)
Comment (yadudoc) [reply docs/quickstart.rst:77]: Added back in.

**Comment by yadudoc** [comment]
@benclifford Could you take a look at the monitoring section, please?

**Pull request merged**